internal/utils/tdl.go

```diff
@@ -198,8 +202,9 @@ func ConvertTwitchLiveChatToTDLChat(path string, channelName string, videoID str
 
 					// ensure that the sliced string equals the emote
 					// sometimes the output of chat-downloader will not include a unicode character when calculating positions causing an offset in positions
-					if slicedEmote != liveCommentEmote.Name {
+					if slicedEmote != liveCommentEmote.Name || message_is_offset == true {
```

Comment:
```suggestion
					if slicedEmote != liveCommentEmote.Name || message_is_offset {
```

To fix the linter you don't need to specify if it's true